packages/offline-payment-integration/e2e/OfflinePaymentMethod.spec.ts

```diff
@@ -0,0 +1,34 @@
+import {
+    PaymentStepAsGuestPreset,
+    test,
+} from '@bigcommerce/checkout/payment-integration-test-framework';
+
+test.describe('Offline payment method', () => {
+    test('`Pay in Store` payment method is working', async ({ assertions, checkout }) => {
+        // Testing environment setup
+        await checkout.use(new PaymentStepAsGuestPreset());
+        await checkout.start('Pay in Store');
```

Comment:
Is this the text on the payment method field?

Reply:
This is just a name for the HAR file.